### PR TITLE
account for none cov model

### PR DESCRIPTION
**Pull request by klaricch**

### CODE REVIEW

File: gnomad_constraint/utils/constraint.py [hunk 1]
```diff
@@ -332,7 +332,7 @@ def apply_models(
     context_ht: hl.Table,
     mutation_ht: hl.Table,
     plateau_models: hl.StructExpression,
-    coverage_model: Tuple[float, float],
+    coverage_model: Optional[Tuple[float, float]],
```
Comment (jkgoodrich) [inline gnomad_constraint/utils/constraint.py:335]:
```suggestion
    coverage_model: Optional[Tuple[float, float]] = None,
```

File: gnomad_constraint/utils/constraint.py [hunk 2]
```diff
@@ -473,7 +473,11 @@ def apply_models(
         hl.case()
         .when(ht.coverage == 0, 0)
         .when(ht.coverage >= high_cov_definition, 1)
-        .default(coverage_model[1] * hl.log10(ht.coverage) + coverage_model[0])
+        .default(
+            (coverage_model[1] * hl.log10(ht.coverage) + coverage_model[0])
+            if coverage_model is not None
+            else 1
```
Comment (jkgoodrich) [inline gnomad_constraint/utils/constraint.py:479]: Can you add a quick note about this to the comment

File: gnomad_constraint/utils/constraint.py [hunk 3]
```diff
@@ -473,7 +473,11 @@ def apply_models(
...
+        )
```
Comment (jkgoodrich) [inline gnomad_constraint/utils/constraint.py:480]: Are globals OK with None now? I think they are, but I know this is an issue I have hade in the past:
```    # Annotate global annotations.
    ht = ht.annotate_globals(
        apply_model_params=hl.struct(
            max_af=max_af,
            pops=pops,
            plateau_models=plateau_models,
            coverage_model=coverage_model,
        )
    )
```

**Review by jkgoodrich — CHANGES_REQUESTED** [review]

File: gnomad_constraint/utils/constraint.py (hunk 3 above)
Comment (klaricch) [reply gnomad_constraint/utils/constraint.py:480]: looks like it's still an issue

**Review by jkgoodrich — APPROVED** [review]
LGTM!

**Pull request merged**